src/api/common/schemas/quality.json

```diff
@@ -1,8 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "quality",
-  "description": "Ratio for incoming/outgoing transit fees represented in billionths. (For example, a value of 500 million represents a 0.5:1 ratio.) As a special case, 0 is treated as a 1:1 ratio.",
-  "type": "integer",
-  "minimum": 0,
-  "maximum": 1000000000
+  "description": "Ratio for incoming/outgoing transit fees. As a special case, 0 is treated as a 1:1 ratio.",
+  "type": "number",
+  "minimum": 0
```

Comment:
I don't know what application level bounds there are, but Quality(In|Out) are serialised as UInt32 fields, so there's at least that: 4.294967295 

> One could argue for consistency

Probably what I would have gone for.